buildflow/runtime/ray_io/base.py

```diff
@@ -136,21 +135,12 @@ def __init__(self, ray_sinks: Dict[str, RaySink]) -> None:
         self._requests = 0
         self._secs = 0
         self._replica_id = utils.uuid()
-        self.throughput_gauge = Gauge(
-            "throughput",
-            description="Current throughput of the actor. Goes up and down.",
-            tag_keys=("actor_name", ),
-        )
-        self.throughput_gauge.set_default_tags(
-            {"actor_name": self.__class__.__name__})
-
-    def update_metrics(self, num_events: int, execution_time_secs: float):
+
+    def update_metrics(self, num_events: int):
         self._num_events += num_events
```

Comment:
Does this ever reset? Python will just keep keep allocating memory instead of spilling over iirc

Reply:
yep it resets whenever we poll it: https://github.com/launchflow/buildflow/blob/main/buildflow/runtime/ray_io/base.py#L167